core/new-gui/src/app/dashboard/component/feature-bar/feature-bar.component.html

```diff
@@ -0,0 +1,15 @@
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
Does this line import font-awesome into the HTML? If it does, it is not necessary since font-awesome is already included in our package.json. The current icons on the navigation bar in the main page is using font-awesome without having this link.